tests/TextTaggerTest.php

```diff
@@ -0,0 +1,193 @@
+        Quisque in erat justo. Praesent eu cursus lectus. Nunc ut egestas mi. Cras venenatis elit eget mi dictum congue. Nam nec venenatis sem. Ut imperdiet metus elementum nisl tincidunt tempus. Vivamus aliquet justo id vestibulum sagittis. Aliquam iaculis consectetur libero. In volutpat, eros at tempor ullamcorper, risus eros viverra libero, nec ultricies metus nibh id ex. Sed tincidunt lacus sollicitudin odio vulputate tincidunt. Duis non sapien et justo egestas facilisis accumsan id diam. Quisque ultrices porttitor dolor a luctus. Quisque vitae est id ligula consequat ultrices. In dolor odio, hendrerit non imperdiet quis, viverra quis arcu.
+        Ut efficitur lacus ac dui volutpat pellentesque. Pellentesque semper ultrices est ac rhoncus. Morbi auctor vitae leo a pellentesque. Pellentesque malesuada nisi eget enim luctus, at volutpat leo aliquam. Sed et neque tempus, fringilla dui eu, ornare velit. Donec vitae lorem vitae tellus consectetur pharetra id quis orci. Ut interdum vulputate odio sit amet vulputate. Nulla id ipsum nulla. Curabitur in lacinia urna. Mauris vitae ultrices diam. Pellentesque in ornare quam. Nullam vehicula, diam non ultrices mollis, nibh diam ornare quam, vel auctor odio enim in tellus. Nullam mattis nulla et enim imperdiet mollis.
+        Duis odio mauris, ullamcorper eget sapien eu, volutpat porta velit. Nullam vel cursus lorem, at facilisis urna. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Etiam maximus sed arcu in viverra. Etiam maximus tellus ut nisl tristique euismod. Etiam imperdiet, mi vitae tempor tempus, dolor turpis faucibus lacus, condimentum consequat quam magna eget mauris. Phasellus eget dignissim odio. Fusce et fermentum urna. Ut aliquam ex nec diam tincidunt imperdiet.
+        Sed vel leo id sem semper ornare eget congue tortor. Quisque lacinia consectetur lorem vitae hendrerit. Proin faucibus tellus sed augue laoreet ultricies. Nam a mattis neque. Vestibulum hendrerit, magna non bibendum mattis, mi mauris ullamcorper elit, sit amet maximus nisl arcu vitae sapien. Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia curae; Integer in neque quis tortor accumsan commodo. Integer pharetra convallis nulla, sit amet hendrerit purus pretium et. Donec volutpat et dui nec venenatis. Pellentesque ullamcorper leo sit amet enim fermentum dictum. Ut elementum aliquet nisi fringilla hendrerit. Integer vestibulum dolor sit amet sem hendrerit, eu facilisis ante rutrum. Nunc nec pellentesque ante.";
+
+        for($i=0; $i<1500; $i++){
+            $tags = $this->textTagger->getTags($str);
+        }
+        $diffTime = time()-$start;
+        $this->assertTrue($diffTime<=6);

```

Comment:
Si tu veut faire un benchmark, il faut s'y prendre d'une autre façon et en général on ne mélange pas avec les tests unitaires.

Surtout que $this->assertTrue($diffTime<=6); ça dépend de ma machine, dison que si je le fait depuis un ordi avec un tout petit processeur qui est bien occupé avec des taches de fond (même un gros processeur, s'il est vraiment très occupé) et bien ton test échouera.

Un bench en soit ça n'échoue pas ça donne des stats en fonction d'une machine et d'un environement